### DISCUSSION TITLE
Changing poll timer

### DISCUSSION THREAD

**Discussion by sandygmaharaj** [camunda-community-hub/pyzeebe#278] [Q&A]
We have built a workflow with 8 service tasks. While testing, we see that it takes around 1 second to execute all of them. Each individual task takes only 5-10 msec. Our use case is to implement near real time work flow, is it possible to reduce time between next task is triggered after previous one finishes?

**Comment by sandygmaharaj** [comment]
Could anybody provide an input or direction on this issue?

**Comment by JonatanMartens** [comment]
Have you taken a look at the `poll_retry_delay` parameter?
See https://pyzeebe.readthedocs.io/en/latest/worker_reference.html#pyzeebe.ZeebeWorker

**Reply by sandygmaharaj** [reply]
Is this value in ms or sec?

**Reply by sandygmaharaj** [reply]
From code:

`            poll_retry_delay (int): The number of seconds to wait before attempting to poll again when reaching max amount of running jobs`

Don't think this will help but thanks anyway for your response.
